sklearn/feature_extraction/text.py

```diff
@@ -1008,6 +1026,8 @@ class TfidfTransformer(BaseEstimator, TransformerMixin):
     Normalization is "c" (cosine) when ``norm='l2'``, "n" (none)
     when ``norm=None``.
 
+    By default, the l2 norm is used.
```

Comment:
How detailed should I go? I'm debating between:
`The norm parameter provides the "how" of how each vector is scaled during normalization, (default is l2)` and what l1 and l2 mean, but the latter feels too detailed for a docstring.